src/licensedcode/data/rules/mit_1351.RULE

```diff
@@ -0,0 +1,9 @@
+---
+license_expression: mit
+is_license_reference: yes
+relevance: 100
+ignorable_urls:
+    - https://opensource.org/licenses/MIT

```

Comment:
add the following:
```
referenced_filename:
   - LICENSE
```
This is how we follow license detections in other files if they are found.